### ISSUE TITLE
I added  one Item  in dependencies.but I don't know how to build,and make it  go into effect

### ISSUE THREAD

**Issue by hasonguo** [flutter/flutter#18909]
I added  one Item  in dependencies, but I don't know how to build, and make it go into effect.
```
dependencies {
  //new added
    implementation 'com.android.support:design:27.1.1'
}
```
I had finished adding it, but my project still not call the package. how to do make it go into effect.

**Comment by ebeem** [comment]
I am not sure what do you mean by 'make it go into effect'
if you add a dependency, you will need to download it through a (just like all other package managers)

to do this, you have to run either
`flutter packages get`
or
`flutter packages upgrade` (I recommend this one)

**this command has to be executed in your project directory**

if you already added, but you want to use it, then you will need to import it and use its API
this should be included in the README of the library you are using

**Comment by zoechi** [comment]
@ebeem `flutter packages get/upgrade` is for dependencies added to `pubspec.yaml`
Do you think it also processes gradle dependencies?
It never produces any related output, therefore I doubt it.

**Comment by ebeem** [comment]
@zoechi my bad, I thought he meant flutter packages
and what u said is probably right

and according to what I have noticed up to now, the gradle dependencies are resolved everytime you run a flutter app. I am not sure about this though, this is the case for me.
here is the output after adding an invalid dependency (a one that does not exist `implementation "com.test.test:3.2.3"` and trying to run the app)

```
Launching lib/main.dart on Nexus 5 in debug mode...
Initializing gradle...
Resolving dependencies...
Running 'gradlew assembleDebug'...

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
> Could not find com.test.test:3.2.3:.
  Required by:
      project :app

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Finished with error: Gradle build failed: 1

```

**Comment by hasonguo** [comment]
gradle --refresh-dependencies, I had tried it.But it still go into effect!
look at  my codes below：
import android.support.v7.app.AppCompatActivity;
public class AndroidActivity extends **AppCompatActivity** {}

Android Studio IDE tells me can't find AppCompatActivity.But I had added dependencies.
HowEver,When I run the flutter project ,it can run successful.
I prefer to tick the "AppCompatActivity" ,it can jump the Android Sources.what Can i do to deal with it
